internal/apiserver/route_get_token_accounts.go

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2021 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"net/http"
+
+	"github.com/hyperledger-labs/firefly/internal/config"
+	"github.com/hyperledger-labs/firefly/internal/i18n"
+	"github.com/hyperledger-labs/firefly/internal/oapispec"
+	"github.com/hyperledger-labs/firefly/pkg/database"
+	"github.com/hyperledger-labs/firefly/pkg/fftypes"
+)
+
+var getTokenAccounts = &oapispec.Route{
+	Name:   "getTokenAccounts",
+	Path:   "namespaces/{ns}/tokens/{type}/pools/{name}/accounts",
```

Comment:
Am I understanding the uniqueness model correctly, that I would expect to see the following route appear in a future PR?
`GET` `namespaces/{ns}/tokens/{type}/pools/{name}/accounts/{index}`

Reply:
(more on this below)

Reply:
Not entirely sure. Currently within a pool, if it is fungible, there is only one token_index (0). If it is non-fungible, there are token_index values starting at 1 and incrementing for each unique token.

A token account is a product of pool + token_index + identity. I'm not sure of the best way to craft a URL to query the balance of an individual account base on those 3 params.

Reply:
I see you already addressed this [elsewhere](https://github.com/hyperledger-labs/firefly/pull/178#discussion_r704675047)